src/internal/connector/onedrive/collection.go

```diff
@@ -498,15 +498,60 @@ func (oc *Collection) populateItems(ctx context.Context, errs *fault.Bus) {
 				metaSuffix = metadata.DirMetaFileSuffix
 			}
 
-			// Fetch metadata for the file
 			itemMeta, itemMetaSize, err = oc.itemMetaReader(
 				ctx,
 				oc.service,
 				oc.driveID,
 				item)
 
 			if err != nil {
-				el.AddRecoverable(clues.Wrap(err, "getting item metadata").Label(fault.LabelForceNoBackupCreation))
```

Comment:
From the logs, the error that the `itemMetaReader` returns is `Item not found`. I think we can just match on that and assume that the item have been deleted instead of trying to get the item again?

Reply:
The `item not found` is coming from [here](https://github.com/alcionai/corso/pull/3485/files#diff-847fd50c281ff780c5a089ba2b3efa718c400a5fc60061c3894298886580fae3R47).

It should actually say `permissions not found`.

Reply:
What I meant was if we get 404 when trying to fetch permissions, can't we just assume that it is because the file was not available and thus just assume the file was deleted without trying to fetch the file again? We could avoid [this](https://github.com/alcionai/corso/blob/100f1eb290a45c047d7789c68a7f7321d147ec8c/src/internal/connector/onedrive/collection.go#L508-L513) completely.

Reply:
>What I meant was if we get 404 when trying to fetch permissions, can't we just assume that it is because the file was not available and thus just assume the file was deleted without trying to fetch the file again

I see. I'm not sure if it's a valid assumption, since permissions is its own dedicated API and can return 404s. 
I don't know whether ignoring a permissions 404 is the right thing to do though. @ryanfkeepers any thoughts?

Reply:
> Since the graph call here [looks up the item by id](https://github.com/alcionai/corso/blob/main/src/pkg/services/m365/api/drive.go#L217), and not the permissions by id, I think we can assume the item was deleted on a 404

This is a good point. thanks! I'll remove the recoverable error for permissions 404 then, just an info log should do fine.